src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -178,6 +189,40 @@ internal DecompressionMethods AutomaticDecompression
             }
         }
 
+        internal CookieUsePolicy CookieUsePolicy
```

Comment:
The idea was to develop like winHttpHandler does. winHttpHandler have 3 value for cookie policy:- IgnoreCookies, UseInternal & UseSpecifiedContainer. Internal is only used by default which relies on winhttp directly. HttpClientHandler.windows UseProxy Boolean sets policy to either IgnoreCookies or UseSpecifiedContainer. Currently IgnoreCookies & UseSpecifiedContainer looks applicable for libcurl. But in case there may come another mode, we have implemented like that. But I see your point, we can change it to Boolean, & if there is a policy requirement comes in later due to extra mode, we shall incorporate the policy.